wpiunits/src/main/java/edu/wpi/first/units/Measure.java

```diff
@@ -237,6 +237,19 @@ default Measure<U> negate() {
     return times(-1);
   }
 
+  /**
+   * Returns the inverse of this measure.
+   *
+   * @return the resulting measure
+   */
+  default Measure<?> inverse() {
+    // Taking the inverse of a temperature breaks and doesn't make sense
```

Comment:
I made this exception after having unexpected behavior writing a test that used Celsius and talking to people on the server, should I simply get rid of the exception? Or is there something to be done because of the whole inverse Celsius and Fahrenheit being weird.

Reply:
The problem with inverting is that temperature units use arbitrary mapping functions, so we can't access the scale and offset values. Refactoring the `Temperature` class to store those values would be possible, but it wouldn't play nicely with deriving units.

I'd be okay with leaving the exception in here, and revisit the temperature case later. But leave a warning in the javadoc so users won't be blindsided